src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ CHANGELOG
  * Added a `InMemoryTransport` to Messenger. Use it with a DSN starting with `in-memory://`.
  * Added `framework.property_access.throw_exception_on_invalid_property_path` config option.
  * Added `cache:pool:list` command to list all available cache pools.
+ * Added a new option to the version strategy called `json_manifest_allow_missing`
+   that allows you to ignore invalid json file.
```

Comment:
What does "invalid" mean? The option name indicates that this is about missing files, but the description makes it look like files with invalid JSON syntax would be ignored too.